lib/php/src/ext/thrift_protocol/config.m4

```diff
@@ -1,25 +1,27 @@
-dnl Copyright (C) 2009 Facebook
-dnl Copying and distribution of this file, with or without modification,
-dnl are permitted in any medium without royalty provided the copyright
-dnl notice and this notice are preserved.
+dnl Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
dreiss's copyright statement says, "provided the copyright notice and this notice are preserved.".  I don't think we should be removing it.  You can move it down to be below the Apache Thrift license, or @jfarrell needs to approve this change.

Reply:
OK, I just add APL declaration to `config.m4`